openupgrade_scripts/scripts/website/14.0.1.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def website_cookie_notice_post_migration(env):
+    cookie_message = env.ref("website.cookie_message", raise_if_not_found=False)
+    if cookie_message:
+        websites = env["website"].search([])
+        websites.write({"cookies_bar": True})
+        openupgrade.logged_query(
+            env.cr,
+            """
+            WITH keys AS (
```

Comment:
Why doing this again?

Reply:
What do you mean again? here we delete all views that have as key the views of 'cookie_message' and 'cookie_message_container'